src/ripple/core/Stoppable.h

```diff
@@ -360,17 +360,9 @@ class RootStoppable : public Stoppable
     bool
     isStopping() const;
 
-    /** Prepare all contained Stoppable objects.
```

Comment:
`prepare` is only ever called right before `start`. There's no need for it to be a separate function, and I think the separation is confusing additional complexity.